priv/test_repo/migrations/1_migrate_all.exs

```diff
@@ -3,23 +3,29 @@ defmodule ExMachina.TestRepo.Migrations.MigrateAll do
 
   def change do
     create table(:users) do
-      add :name, :string
-      add :admin, :boolean
-      add :net_worth, :decimal
+      add(:name, :string)
+      add(:admin, :boolean)
+      add(:net_worth, :decimal)
     end
 
+    create table(:publishers) do
+      add(:pub_number, :string)
+    end
+
+    create(unique_index(:publishers, [:pub_number]))
```

Comment:
Added this uniqueness constraint to properly test the scenario in ecto where `insert_pair(:account, publisher: build(:publisher))` would raise a constraint error.